pack/side/hotel_encounter.json

```diff
@@ -1,62 +1,934 @@
+		"shroud": 4,
+		"text": "[action]: Test [intellect] (3). If you succeed, move any number of clues controlled by investigators at this location to Sinister Solution (if it is in play).",
+		"traits": "Crime Scene.",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_flavor": "A deathly chill emanates from beneath the door to Room 245",
+		"back_illustrator": "Yoann Boissonnet",
+		"clues": 1,
+		"code": "84017",
+		"encounter_code": "murder_at_the_excelsior_hotel",
+		"encounter_position": 19,
+		"faction_code": "mythos",
+		"flavor": "The room beyond is immaculate, seemingly untouched in quite some time.",
+		"illustrator": "Dual Brush Studios",
+		"name": "Room 245",
+		"pack_code": "hotel",
+		"position": 17,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "[action]: Test [intellect] (3). Only the top card of your discard pile may be committed to this skill test (place it on the bottom of your deck after this test resolves). If you succeed, move any number of clues controlled by investigators in Room 245 to Time-Worn Locket (if it is in play).",
+		"traits": "Crime Scene.",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_illustrator": "Yoann Boissonnet",
+		"back_text": "The door to the Office is locked. You cannot enter the Office unless you control the Manager's Key.\nBasement gains: \"[action]: Test [agility] (3) to attempt to pick the lock. If you succeed, reveal Office and immediately move to it.\"",
+		"clues": 2,
+		"code": "84018",
+		"encounter_code": "murder_at_the_excelsior_hotel",
+		"encounter_position": 20,
+		"faction_code": "mythos",
+		"flavor": "The manager of the Excelsior is obviously a busy person - newspaper clippings, strange documents, and keys of all sorts are scattered about.",
+		"illustrator": "Dual Brush Studios",
+		"name": "Office",
+		"pack_code": "hotel",
+		"position": 18,
+		"quantity": 1,
+		"shroud": 3,
+		"text": "[action]: Test [intellect] (0). For each point you succeed by, you may move 1 clue controlled by an investigator in the Office to Manager's key (if it is in play).",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_flavor": "The hum of machinery and the dim lighting make your hair stand on end. Exploring the basement will be difficult. Amidst the din, you think you hear footsteps echoing your own, just a few paces behind.",
+		"back_illustrator": "Yoann Boissonnet",
+		"clues": 1,
+		"code": "84019",
+		"encounter_code": "murder_at_the_excelsior_hotel",
+		"encounter_position": 21,
+		"faction_code": "mythos",
+		"illustrator": "Mark Molnar",
+		"name": "Basement",
+		"pack_code": "hotel",
+		"position": 19,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "Veiled.",
+		"traits": "[action]: Test [intellect] (1). If you succeed, move a [[Humanoid]] enemy at any location once toward the Basement.\n[reaction] After you defeat an enemy in the Basement: Move any number of clues controlled by investigators at the Basement to Tome of Rituals (if it is in play).",
```

Comment:
Looks like the text wound up in traits.  Traits should also be Crime Scene instead of Veiled.